File: src/client/src/app/core/enums/RequestStatus.ts
export enum RequestStatus {
    Pending = 0,
    Rejected = 1,
    Approved = 2,
    InProgress = 3
}
export const RequestStatusMapping: Record<RequestStatus, string> = {
    [RequestStatus.Approved]: "Approved",
    [RequestStatus.Rejected]: "Rejected",
    [RequestStatus.Pending]: "Pending",
    [RequestStatus.InProgress]: "In Progress",
};

File: src/server/Shared/Shared.DTOs/Enums/RequestStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FluentPOS.Shared.DTOs.Enums
{
    public enum RequestStatus
    {
        Pending = 0,
        Rejected = 1,
        Approved = 2,
        InProgress = 3
    }
}